source/installation/add-ssl.rst

```diff
@@ -61,7 +63,56 @@ Intermediate certificate
       For documentation on SSL directives please see:
       https://httpd.apache.org/docs/2.4/mod/mod_ssl.html
 
-#. Restart the Apache service for the changes take effect.
+2. Update CA (Dex Users only)
+`````````````````````````````
+
+Dex users may encounter issues with SSL certificates like::
+
+  remote error: tls: unknown certificate authority
+
+If this is the case, you need to ensure that their certificate authority (CA) is
+in the system trust store and that your CA certificates are up to date.
+
+First, try updating your CA certificates.  This could especially happen when
+you have a `Let's Encrypt`_ and your machine does not know it's a certificate
+authority.
+
```

Comment:
I think this is worded incorrectly.  Here is possible suggestion:
```suggestion
you have a `Let's Encrypt`_ and your machine does not know about that certificate
authority.

```

The certificate authority is LetsEncrypt.